### PR TITLE
Emoji support + template fixes

### PR DESCRIPTION
**Pull request by bedroesb**
I changed in the templates the metadata for: `folder: RDM_cycle` because this is a constant and 
`search: false` to disable that the searchbar will look into the template pages

### CODE REVIEW

File: pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ keywords: [Keyword 1,Keyword 2, Keyword n]
 summary: Fill in your summary or delete this property when not needed
 permalink: TEMPLATE_RDM_cycle.html
 folder: RDM_cycle
+search: false
```
Comment (floradanna) [inline pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md:7]: what it this "search" ? What's for?

Comment (bedroesb) [reply pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md:7]: it is to disable that that page will end up in the search results. This line will thus need to be removed when the template is copied ...

Comment (bedroesb) [reply pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md:7]: If you think it is not a problem the searchbar searches in the Template pages, we can leave the line out off course

Comment (floradanna) [reply pages/RDM_cycle/TEMPLATE_RDM_cycle_WP1.md:7]: It makes sense to disable that that page from search, indeed.

**Pull request merged**